releasenotes/notes/ignore_stopiteration_exception-43806201b7fc8005.yaml

```
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    wsgi: Previously ``StopIteration`` exceptions which are expected to be raised after closing a websocket connection,
```

This isn't a wsgi specific fix. It should impact all spans.

```suggestion
    tracing: Ensures the status of a Span is not set to error when a``StopIteration`` exception is raised
```